pkg/policies/opa/rego/docker/docker_run/AC_DOCKER_0033json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "yumInstallWithoutVersion",
+    "file": "yumInstallWithoutVersion.rego",
+    "policy_type": "docker",
+    "resource_type": "docker_run",
+    "template_args": {
+        "prefix": "",
+        "suffix": ""
+    },
+    "severity": "MEDIUM",
+    "description": "Ensure package version is specified to avoid failures",
+    "reference_id": "AC_DOCKER_0031",
+    "category": "Infrastructure Security",
+    "id": "AC_DOCKER_0031",
```

Comment:
Duplicate id and reference id,  `pipInstallKeepingCachedPackages` policy also have the same id. Typo in filename `pkg/policies/opa/rego/docker/docker_run/AC_DOCKER_0033json`